lib/system.nim

```diff
@@ -2592,6 +2592,27 @@ proc substr*(s: openArray[char]): string =
   for i, ch in s:
     result[i] = ch
 
+proc substr*(s: openArray[char] , first, last: int): string =
```

Comment:
This is not used afaict.

Reply:
`runeSubstr` uses it once, so seemed sensible to port it over to openarray, for an instance where someone wants the same behaviour for openarray in the future.

Reply:
So move it to unicode.nim and don't export it for the time being.